core/src/main/java/hudson/ExtensionListView.java

```diff
@@ -108,12 +108,12 @@ public boolean remove(Object o) {
             }
 
             @Override
-            public void add(T t) {
+            public synchronized void add(T t) {
                 storage().add(t);
             }
 
             @Override
-            public boolean remove(T t) {
+            public synchronized boolean remove(T t) {
```

Comment:
Likely incorrect (the delegate has its own synchronization semantics). Anyway all callers of `createCopyOnWriteList` are deprecated. Best to deprecate this method as well, and just suppress the EP warnings here.